### PR TITLE
Adding minimal samples (for the doc)

### PR DESCRIPTION
**Pull request by jmprieur**
Adding minimal daemon samples:
- daemon app calling msgraph
- daemon app calling downstream web API

### CODE REVIEW

File: tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <TargetFrameworks>net7.0</TargetFrameworks>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\minimal-web-api\WeatherForecast.cs" Link="WeatherForecast.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(UseWip)' == ''">
```
Comment (jmprieur) [inline tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj:14]: We can either build from NuGet / artifact feed, or from the solution.

Would it make sense to have an environment variable in the Directory.Build.Props with the version of Id.Web to test?

File: src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory.cs [hunk 2]
```diff
@@ -61,8 +63,15 @@ protected TokenAcquirerFactory()
...
 
         /// <summary>
-        /// Get the default instance.
+        /// Get the default instance of a token acquirer factory. Use this method, for instance, if you have an ASP.NET OWIN application
+        /// and you want to get the default instance of the OwinTokenAcquirerFactory.
         /// </summary>
+        /// /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[ConvertType](~/../tests/DevApps/aspnet-mvc/OwinWebApp/App_Start/Startup.Auth.cs?highlight=22)]
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquirerFactory.cs:72]: `~` is the path containing the docfx.json file

File: tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj (hunk 1 above)
Comment (jennyf19) [reply tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj:14]: Yes, but you dont need to add it to this PR.

File: tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <PackageReference Include="Microsoft.Identity.Web.DownstreamRestApi" Version="2.2.1" />
```
Comment (jennyf19) [inline tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj:15]: `Rest` api should not be a thing anymore :(

File: tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(UseWip)' == 'true'">
+    <ProjectReference Include="..\..\..\..\src\Microsoft.Identity.Web.DownstreamRestApi\Microsoft.Identity.Web.DownstreamRestApi.csproj" />
```
Comment (jennyf19) [inline tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/DaemonConsoleCallingDownstreamApi.csproj:19]: Same here w/wrong reference.

File: tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/Program.cs [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+var tokenAcquirerFactory = TokenAcquirerFactory.GetDefaultInstance();
+tokenAcquirerFactory.Services.AddDownstreamApi("MyApi", 
+    tokenAcquirerFactory.Configuration.GetSection("MyWebApi"));
+var sp = tokenAcquirerFactory.Build();
+
+var api = sp.GetRequiredService<IDownstreamApi>();
+var result = await api.GetForAppAsync<IEnumerable<WeatherForecast>>("MyApi");
+Console.WriteLine($"result = {result?.Count()}");
+
```
Comment (jennyf19) [inline tests/DevApps/daemon-app/daemon-console-calling-downstreamApi/Program.cs:14]: nice

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Pull request merged**